apps/web/src/pages/invites/MembersInvitePage.tsx

```diff
@@ -0,0 +1,105 @@
+import { Form } from 'antd';
+import { useMutation, useQuery } from 'react-query';
+import styled from 'styled-components';
+import { MemberStatusEnum } from '@notifire/shared';
+import * as capitalize from 'lodash.capitalize';
+import { Avatar, Container, Divider, Group, Text } from '@mantine/core';
+import PageHeader from '../../components/layout/components/PageHeader';
+import { getOrganizationMembers, inviteMember } from '../../api/organization';
+import PageContainer from '../../components/layout/components/PageContainer';
+import { Button, Input, Tag } from '../../design-system';
+import { Invite } from '../../design-system/icons';
+import useStyles from '../../design-system/config/text.styles';
+
+export function MembersInvitePage() {
+  const [form] = Form.useForm();
+  const { classes } = useStyles();
+
+  const {
+    data: members,
+    isLoading: loadingMembers,
+    refetch,
+  } = useQuery<any[]>('getOrganizationMembers', getOrganizationMembers);
+  const { isLoading: loadingSendInvite, mutateAsync: sendInvite } = useMutation<
+    string,
+    { error: string; message: string; statusCode: number },
+    string
+  >((email) => inviteMember(email));
+
+  async function onSubmit({ email }) {
+    if (!email) return;
+
+    await sendInvite(email);
+    await refetch();
+    form.resetFields(['email']);
+  }
+
+  return (
+    <PageContainer>
+      <PageHeader
+        title="Team Members"
+        actions={
+          <Form onFinish={onSubmit} form={form}>
+            <Group align="center" spacing={10}>
+              <Form.Item name="email" rules={[{ required: true }]} style={{ marginBottom: 0 }}>
+                <Input required data-test-id="invite-email-field" placeholder="Invite user by email" />
+              </Form.Item>
+              <Button submit icon={<Invite />}>
+                Invite
+              </Button>
+            </Group>
+          </Form>
+        }
+      />
+
+      <Container fluid mt={15} ml={5}>
+        {members?.map((i) => {
```

Comment:
I think we should add a LoadingOverlay here, wdyt? 
You could use the `loadingMembers`  you get from the useQuery